src/shared/feature-flags/flags-status-table.tsx

```diff
@@ -65,21 +67,25 @@ export const FlagsStatusTable: React.FC<Props> = ({ data }) => {
         manage the environment-specific targeting and rollout rules.
       </p>
       <ConfigProvider
-        renderEmpty={() => (
-          <Space direction="vertical">
-            <Typography.Title level={5} type="secondary">
-              {t("feature_flags.not_found_title")}
-            </Typography.Title>
-            <Typography.Text type="secondary">
-              {t("feature_flags.not_found_description")}
-            </Typography.Text>
-            <NewFlagController newFlagLabel="CREATE FLAG" />
-          </Space>
-        )}
+        renderEmpty={() =>
+          isLoading ? (
```

Comment:
You can wrap Space directly using Skeleton.
Like https://codesandbox.io/s/w2k24